test/unit/test_util.c

```diff
@@ -113,6 +113,42 @@ bool test_autonames(void) {
 	mu_end;
 }
 
+bool test_file_slurp(void) {
+
+#ifdef __WINDOWS__
+#define S_IRWXU _S_IREAD | _S_IWRITE
+#endif
+
+	char* test_file =  "./empty_file";
+	size_t s;
+	char* content;
+	char* some_words = "some words";
+
+	close(open(test_file, O_CREAT, S_IRWXU));
+
+	content = r_file_slurp(test_file, &s);
+
+	mu_assert_eq (s, 0, "size should be zero");
+	mu_assert_eq (strlen(content), 0, "returned buffer should be empty");
+
+	free(content);
+
+	int f = open(test_file, O_CREAT|O_WRONLY, S_IRWXU);
```

Comment:
check for the return value of open

Reply:
the file wasnt removed, so o_creat shuoldnt be correct tob e used here

Reply:
done (both)